src/analyzer/vulnerabilities/arbitrary_from_in_transferfrom.rs

```diff
@@ -0,0 +1,124 @@
+use std::collections::HashSet;
+
+use solang_parser::pt::{self, Expression, Loc};
+use solang_parser::{self, pt::SourceUnit};
+
+use crate::analyzer::ast::{self, Target};
+
+pub fn _arbitrary_from_in_transferfrom_vulnerability(source_unit: SourceUnit) -> HashSet<Loc> {
```

Comment:
Can you remove the leading `_` from the function name?

Reply:
Done